contentgrid-spring-data-rest/build.gradle

```diff
@@ -35,21 +35,30 @@ dependencies {
     implementation 'jakarta.servlet:jakarta.servlet-api'
 
     implementation 'org.springframework.data:spring-data-rest-webmvc'
+    implementation 'com.github.paulcwarren:spring-content-rest'
 
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'org.springframework.boot:spring-boot-starter-web'
     testImplementation 'jakarta.transaction:jakarta.transaction-api'
 
+
+
     testFixturesAnnotationProcessor 'com.querydsl:querydsl-apt::jpa'
     testFixturesAnnotationProcessor 'jakarta.persistence:jakarta.persistence-api'
 
     testFixturesImplementation platform(project(':contentgrid-spring-boot-platform'))
 
     testFixturesApi 'org.springframework.data:spring-data-jpa'
+    testFixturesApi 'com.github.paulcwarren:spring-content-commons'
```

Comment:
These are only used for annotations on the model and repositories.

I think they don't have to be an API dependency then, I think annotations are not part of the exposed API.